JSDemos/Demos/Resizable/Overview/description.md

```diff
@@ -1 +1,21 @@
-The Resizable is a UI component that enables its content to be resizable in the UI.
\ No newline at end of file
+The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.
+
+Configure the following properties to specify the boundaries to which you can drag the handles:
+
+- [minWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minWidth)    
+The lower width boundary.
```

Comment:
I'm not sure you even need these descriptions. The property names are self explanatory. And the expressions like "lower width boundary" is just a way to say "minimum width" while trying to avoid repetition. But the phrase doesn't sound right to me. I would simply leave the list of properties without any additional comments.